front/lib/auth.ts

```diff
@@ -204,6 +206,7 @@ export async function getUserFromSession(
   const memberships = await Membership.findAll({
     where: {
       userId: user.id,
+      role: { [Op.in]: ["admin", "builder", "user"] },
```

Comment:
As per IRL: when you revoke someone their membership becomes revoked and here we should not load the workspace for which the user have been revoked (so that he won't have the option to pick a workspace in their UI)